nixos/modules/services/x11/xautolock.nix

```diff
@@ -45,28 +55,80 @@ in
         };
 
         notifier = mkOption {
-          default = "notify-send 'Locking in 10 seconds'";
-          type = types.string;
+          default = null;
+          example = "notify-send 'Locking in 10 seconds'";
+          type = types.nullOr types.string;
 
           description = ''
             Notification script to be used to warn about the pending autolock.
           '';
         };
+
+        killer = mkOption {
+          default = null; # default according to `man xautolock` is none
+          example = "systemctl suspend";
+          type = types.string;
```

Comment:
must be `types.nullOr types.string`.
I left this one empty and I got ``error: The option value `services.xserver.xautolock.killer' in `/home/ma27/Projects/nixpkgs/nixos/modules/services/x11/xautolock.nix' is not a string.``